transmart-rest-api/grails-app/services/org/transmartproject/rest/HypercubeDataSerializationService.groovy

```diff
@@ -43,7 +43,8 @@ class HypercubeDataSerializationService implements DataSerializer {
     void writeClinical(Format format,
                        MultiDimConstraint constraint,
                        User user,
-                       OutputStream out) {
+                       OutputStream out,
+                       boolean includeMeasurementDateColumn = false) {
```

Comment:
This service does not use this flag. It would the best to make it specific to `SurveyTableViewDataSerializationService`.

Reply:
but SurveyTableViewDataSerializationService and HypercubeDataSerializationService implement the same interface

Reply:
You could add an optional `Map options`, that contains format or view specific options.

Reply:
But then we don't have static typing for such options.